src/contoroller/category/category.js

```diff
@@ -7,36 +7,19 @@ class CategoryController {
     this.User = User;
   }
 
-  //** OK **//
   async create(req, res) {
     //Validte received data to create a new category
     const { error } = this.validateCategory(req.body);
     if (error) return res.status(400).send(error.message);
-    //Find the authorized user
-    const user = await this.User.findOne({ where: { id: req.user.id } });
-
-    //Prevent duplication in category table for a user
-    const existingCategory = await this.Category.findOne({
-      where: {
-        userId: user.id,
-        name: req.body.name,
-        type: req.body.type,
-      },
-    });
-    if (existingCategory) {
-      return res.statu(409).send("this category has been defined");
-    }
-
     //Create a new category with given data
```

Comment:
What happens if there is a duplicate name+userId error happens here? Shouldn't you take care of that?

Reply:
for this I have ran 2 migration and created constraints in accounts and categories  models.
in account 2 column "userId", and "name"
in categories 3 column "userId", "name", and also "type"
I tested them and db do not accept ducplication now, but I do not know how to handle the error and send response to such requests
I checked the server container and posgres container but non of them send me an error!!!  postman just hang in there